manifest.jps

```diff
@@ -0,0 +1,57 @@
+{
+	"jpsType": "install",
+	"application": {
+		"name": "TeamCity",
+		"homepage": "https://www.jetbrains.com/teamcity/",
+		"logo": "http://goo.gl/g89D9C", 
+		"description": "TeamCity is a Java-based build management and continuous integration server",
+		"success": {
+			"text": "Instance address: <a href='${env.url}' target='_blank'>${env.url}</a> </br></br>To add custom domain name for your TeamCity installation follow the steps described in our <a href='http://docs.jelastic.com/custom-domains' target='_blank'>documentation</a>",
+			"email": "Instance address: <a href='${env.url}' target='_blank'>${env.url}</a> </br></br>To add custom domain name for your TeamCity installation follow the steps described in our <a href='http://docs.jelastic.com/custom-domains' target='_blank'>documentation</a>"
+		},
+		"env": {
+			"topology": {
+				"nodes": [{
+					"displayName": "TeamCity server",
+					"cloudlets": 16,
+					"nodeType": "docker",
+					"nodeGroup": "cp",
+					"docker": {
+						"image": "sych74/teamcity",
```

Comment:
You can replace my image by your own, but to make it work you need first to accept changes I sent you in my pull request above